recipes/openvino-telemetry/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "openvino-telemetry" %}
+{% set version = "2022.3.0" %}
```

Comment:
Latest version is 2023.1.0 but that one seems to be missing a sdist on PyPI. Can you open an issue upstream to request one?